app/Jobs/SendMetaConversionEventJob.php

```
@@ -141,4 +151,53 @@ public function failed(\Throwable $exception): void
+                .(isset($result['status_code']) ? " (HTTP {$result['status_code']})" : '')
+            : "Meta Conversion event \"{$this->eventName}\" failed: ".($result['error'] ?? 'unknown error');
+
+        try {
+            DealAutomationLog::create([
+                'company_id' => $this->subject->company_id,
+                'deal_id' => $this->subject instanceof Deal ? $this->subject->id : null,
+                'lead_id' => $this->subject instanceof Lead ? $this->subject->id : null,
+                'automation_id' => $this->origin['automation_id'] ?? null,
+                // Joins the automation run that queued it. A stage-trigger event
+                // has no run to join, so it gets its own single-step id.
+                'run_id' => $this->origin['run_id'] ?? 'meta-'.Str::uuid(),
```

PHPStan (2.2.8)

[error] Call to an undefined static method Illuminate\Support\Str::uuid().


(staticMethod.notFound)